ethereum/datastructures/src/main/java/tech/pegasys/artemis/datastructures/networking/libp2p/rpc/StatusMessage.java

```diff
@@ -57,7 +57,7 @@ public static StatusMessage createPreGenesisStatus() {
         UnsignedLong.ZERO);
   }
 
-  private static Bytes4 createPreGenesisForkDigest() {
+  public static Bytes4 createPreGenesisForkDigest() {
```

Comment:
Do we need this now that we only start the network post-genesis?

Reply:
No. Switching back.